packages/contracts/package.json

```
@@ -38,7 +38,7 @@
     "@prb/test": "^0.6.4",
     "@towns-protocol/prettier-config": "workspace:^",
     "@wagmi/cli": "^2.2.0",
-    "account-abstraction": "github:eth-infinitism/account-abstraction",
+    "account-abstraction": "github:eth-infinitism/account-abstraction#cc3893bcaf2272c163ce89d5eb9eadb8e6b52db7",
```

```suggestion
    "account-abstraction": "github:eth-infinitism/account-abstraction#v0.8.0",
```
A release tag is more readable.

---

nice call, I'm pinning to v0.7, since it was the version that we we're using. We can update in another PR